### DISCUSSION TITLE
can't invite any collaborators in private repo

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [community/community#19341] [Repositories]
https://user-images.githubusercontent.com/44552983/175499917-cdf792de-dd0d-4666-892f-38b500244ab0.mov

**Comment by melvincwng** [comment]
@MuttakinHasib, thanks for highlighting this to GitHub Community.

I have addressed this issue encountered by another member in [this thread](https://github.com/github-community/community/discussions/19330).

Basically, my suggestion would be to raise a ticket to Github Support over [here](https://support.github.com/contact/account) and let them investigate this issue.

Hope this helps.

**Reply by melvincwng** [reply]
@MuttakinHasib, if this reply has helped you, please 'Mark as answer' so others can see that this issue has been addressed.

Have a great day ahead, thanks :)
